src/test/java/com/systelab/seed/patient/PatientResourceTest.java

```diff
@@ -214,24 +217,21 @@ public void testGetPatient() {
                 .then().assertThat().statusCode(200)
                 .extract().as(Patient.class);
         Assertions.assertNotNull(patientRetrieved, "Patient not retrieved");
-        if (patientRetrieved != null) {
-            TestUtil.checkField("Name", "John", patientRetrieved.getName());
-            TestUtil.checkField("Surname", "Burrows", patientRetrieved.getSurname());
-            TestUtil.checkField("Email", "jburrows@werfen.com", patientRetrieved.getEmail());
 
-            TestUtil.checkField("Medical Number", "123456", patientRetrieved.getMedicalNumber());
-            TestUtil.checkField("DOB", LocalDate.of(1948, 8, 11), patientRetrieved.getDob());
-        }
+        if (patientRetrieved != null)
+            checkPatientData(patientCreated, patientRetrieved);
+
     }
 
     @Description("Get an Excel file with patients")
     @Test
     public void testGetPatientsExcel() {
```

Comment:
testGetPatientsExcel() must verify the content of the body. It currently verifies the statusCode